source/installguide/management-server/_pkg_repo.rst

```diff
@@ -97,13 +97,13 @@ repository to the file (replace "trusty" with "xenial" or "bionic" if it is the
 
 .. parsed-literal::
 
-   deb http://download.cloudstack.org/ubuntu focal |version|
+   deb http://download.cloudstack.org/ubuntu xenial |version|
```

Comment:
This PR was initially replacing **trusty** with **xenial**, but a commit merged recently added **focal**. Can you please keep **focal** instead of **xenial**?

Reply:
agree

Reply:
done, i think this was the conflit